modules/ROOT/pages/index.adoc

```diff
@@ -1 +1,58 @@
-= Agent Visualizer
+= Agent Visualizer Overview
+
+Agent Visualizer provides a detailed visual graph so you can see, understand, and interact with your enterprise's evolving network of agents, Model Context Protocol (MCP) Servers, and LLMs. Agent Visualizer gives you insight into hidden agent interactions, governance gaps, and performance bottlenecks. Agent Visualizer offers complete situational awareness of your agentic ecosystem. 
+
+With Agent Visualizer, you can:
+
+* Get a clear view of connections and interactions among all your agents, clients, and Model Context Protocol (MCP) servers.
+* Understand data movement and dependencies at a glance by seeing which systems are calling your agents and what those agents are calling in turn.
+* Detect potential issues proactively.
+* Optimize agent orchestration for performance, trust, and scale. 
+
+Agent Visualizer is an intuitive user interface that helps you:
+
+ * Gain full visibility into your agent network by showing all connected agents in one network map, with clear relationships and direction of calls, gaining complete situational awareness of your agentic ecosystem. 
```

Comment:
I think this is switching between "you" (gain visibility) and the UI (showing all agents).

Reply:
This is a very complicated sentence. Is the point "full visibility" or "complete situational awareness"?